ch_tools/common/commands/clean_object_storage.py

```diff
@@ -75,7 +75,7 @@ def clean(
 
         execute_query(
             ctx,
-            f"CREATE TABLE IF NOT EXISTS {listing_table} (obj_path String, obj_size UInt64) ENGINE MergeTree ORDER BY obj_path SETTINGS storage_policy = '{config['storage_policy']}'",
+            f"CREATE TABLE IF NOT EXISTS {listing_table} (obj_path String, obj_size UInt64) ENGINE ReplicatedMergeTree('/clickhouse/tables/{{shard}}/{listing_table}', '{{replica}}') ORDER BY obj_path SETTINGS storage_policy = '{config['storage_policy']}'",
```

Comment:
Suggest to setup table on all cluster hosts, i.e. to add ON CLUSTER. And also replace DROP TABLE with TRUNCATE and do it after CREATE TABLE.

Reply:
done